Stable_Diffusion_WebUi_Altryne.ipynb

```diff
@@ -1,45 +1,10 @@
 {
-  "nbformat": 4,
-  "nbformat_minor": 0,
-  "metadata": {
-    "colab": {
-      "name": "Stable Diffusion WebUi - Altryne",
-      "provenance": [],
-      "collapsed_sections": [
-        "4eodrBxeCgOW",
```

Comment:
Too bad this got deleted, I think all sections should be collapsed

Reply:
I meant to fix that. Oops.